runtime/parachains/src/paras_inherent.rs

```diff
@@ -24,6 +24,7 @@
 use crate::{
 	disputes::DisputesHandler,
 	inclusion,
+	paras::OnChainVotes,
```

Comment:
yeah, reaching directly into the storage of another module is not best practice. As mentioned in https://github.com/paritytech/polkadot/pull/3924/files#r720684852 this storage item should be in `ParasInherent`.